src/modules/interfaces.ts

```diff
@@ -530,45 +635,17 @@ export class ModuleBase {
       install: migrationClass.prototype.up,
       remove: migrationClass.prototype.down,
     };
-    if (this.sql?.afterInstallSqlPath) {
-      try {
-        const sql = fs.readFileSync(`${this.dirname}/${this.sql.afterInstallSqlPath}`, 'utf8');
-        this.migrations.afterInstall = async (q: QueryRunner) => {
-          await q.query(sql);
-        };
-      } catch (e) {
-        logger.warn(`Unable to read file ${this.dirname}/${this.sql.afterInstallSqlPath}`);
-      }
-    } else {
-      // If no path specified, try to get the default
-      try {
-        const sql = fs.readFileSync(`${this.dirname}/sql/after_install.sql`, 'utf8');
-        this.migrations.afterInstall = async (q: QueryRunner) => {
-          await q.query(sql);
-        };
-      } catch (_) {
-        /** Don't do anything if the default file is not there */
-      }
+    const afterInstallMigration = customAfterInstallSql + rpcAfterInstallSql;
+    const beforeUninstallMigration = rpcBeforeUninstallSql + customBeforeUninstallSql;
```

Comment:
Should there be a `"\n"` added in between these two to make sure concatenation doesn't produce unparseable SQL?

Reply:
I make sure of that by adding break lines in the strings with the template strings syntax ``

Reply:
got it